src/configs/index.ts

```diff
@@ -23,8 +23,8 @@ export function mysqlConfig(options?: IMysqlConfig): string {
 export function mongoConfig(options?: IMongoConfig): string {
     try {
         return new RegExp(/^(?:(25[0-5]|2[0-4][0-9]|1[0-9][0-9]|[1-9][0-9]|[0-9])(\.(?!$)|$)){4}$/).test(options.mongohost)
-            ? `mongodb://${options.mongousername.trim()}:${options.mongopassword.trim()}@${options.mongohost.trim()}/${options.mongodatabase.trim()}?retryWrites=true`
-            : `mongodb+srv://${options.mongousername.trim()}:${options.mongopassword.trim()}@${options.mongohost.trim()}/${options.mongodatabase.trim()}?retryWrites=true`;
+            ? `mongodb://${options.mongousername.trim()}:${options.mongopassword.trim()}@${options.mongohost.trim()}/${options.mongodatabase.trim()}?authSource=admin&readPreference=primary&retryWrites=true&w=majority&ssl=false`
```

Comment:
Auth source is not always admin, some users authenticate to different databases depending on type of user